pkg/registry/registry.go

```diff
@@ -57,17 +57,30 @@ func HandlerResultSuccess() HandlerResult {
 // Registry
 
 type Registry struct {
-	SyscallHandler map[string]HandlerFunc
+	syscallHandler map[string]HandlerFunc
+	defaultHandler HandlerFunc
```

Comment:
Isn't it simpler to make this public and just assign it instead of having a setter? I'm not fan of setters, but if you prefer this way... it is ok :)